sdks/java/core/src/main/java/org/apache/beam/sdk/io/FileBasedSink.java

```diff
@@ -135,6 +135,28 @@ public void validate(PipelineOptions options) {}
   @Override
   public abstract FileBasedWriteOperation<T> createWriteOperation(PipelineOptions options);
 
+  @Override
+  public void populateDisplayData(DisplayData.Builder builder) {
+    String fileNamePattern = String.format("%s%s%s",
```

Comment:
add `super` even if empty as a style guide. If `super` data is not desired, add a comment saying why. ?

(For people who copy and paste later.)